src/components/UserManagedList.js

```diff
@@ -199,14 +200,25 @@ export default class UserManagedList extends SystemManagedList {
     const list = coordinates.map(entry => entry.toString())
     const notices = await getNotices(token, list, options.renderer, options.options)
     const { summary, content } = notices
-    const message = `Created Notices file with ${summary.total} entries, ${summary.missing} missing entries, and ${
-      summary.warnings.noLicense.length
-    } missing licenses`
-    if (summary.missing || summary.warnings.noLicense.length) uiWarning(dispatch, message)
+    const message = `Created Notices file with ${summary.total} entries`
+    if (summary.warnings.noLicense.length) uiWarning(dispatch, this._renderNoticeProblem(message, summary))
     else uiInfo(dispatch, message)
     return saveAs(new File([content], `${options.filename}`))
   }
 
+  _renderNoticeProblem(baseMessage, summary) {
+    return (
+      <div>
+        {`${baseMessage}, and ${summary.warnings.noLicense.length} missing licenses`}
+        <div>
+          <Button bsStyle="warning" onClick={() => this.onFilter({ 'licensed.declared': 'absence' }, true)}>
```

Comment:
it would be great to get `{ 'licensed.declared': 'absence' }` to  include NONE and NOASSERTION.

opened https://github.com/clearlydefined/website/issues/428